### PR TITLE
profile: is-editable check

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

File: tests/unit-test/profile.ts [hunk 1]
```diff
@@ -2,86 +2,124 @@ import { CONFIG, wait } from '../common_config'
 import { installAgents } from '../installAgents'
```
Comment (zo-el) [inline tests/unit-test/profile.ts:2]: This file is only linting changes

File: tests/installAgents.ts [hunk 2]
```diff
@@ -1,82 +1,98 @@
-
```
Comment (zo-el) [inline tests/installAgents.ts:1]: this file is only lint updates

File: zomes/profile/src/lib.rs [hunk 3]
```diff
@@ -65,10 +55,40 @@ fn validate(op: Op) -> ExternResult<ValidateCallbackResult> {
...
+}
+
+/// Deserialize properties into the Props expected by this zome
+pub fn is_editable_sb(encoded_props: &SerializedBytes) -> bool {
+    let maybe_props = Props::try_from(encoded_props.to_owned());
+    if let Ok(props) = maybe_props {
+        return props.is_editable_profile;
+    }
+    true
+}
```
Comment (robbiecarlton) [inline zomes/profile/src/lib.rs:94]: not feedback for this PR, just noticing this is a horrible amount of work to have to do to get a boolean dna property 😅

File: tests/unit-test/profile_not_editable.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import { installAgents } from '../installAgents'
+
+module.exports = (orchestrator) => {
+	orchestrator.registerScenario('test profile zomes', async (s, t) => {
+		// spawn the conductor process
+		const [conductor] = await s.players([CONFIG])
+		let [alice_happ, bobbo_happ] = await installAgents(
+			conductor,
+			['alice', 'bobbo'],
+			false
```
Comment (robbiecarlton) [inline tests/unit-test/profile_not_editable.ts:11]:
```suggestion
			false // is_editable_profile dna propery
```

File: tests/unit-test/profile.ts [hunk 5]
```diff
@@ -2,86 +2,124 @@ import { CONFIG, wait } from '../common_config'
...
 
 module.exports = (orchestrator) => {
+ orchestrator.registerScenario('test profile zomes', async (s, t) => {
+  // spawn the conductor process
+  const [conductor] = await s.players([CONFIG])
+  let [alice_happ, bobbo_happ] = await installAgents(conductor, [
+   'alice',
+   'bobbo',
+  ])
```
Comment (robbiecarlton) [inline tests/unit-test/profile.ts:11]: how does passing `is_editable_profile` as undefined here work? I'm missing where it defaults to true. Tests are passing, so obviously it works, but I'd like to understand how.

File: tests/installAgents.ts (hunk 2 above)
Comment (robbiecarlton) [reply tests/installAgents.ts:1]: not true! :)

**Pull request merged**